src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -148,7 +148,8 @@ def default_api_version(self):
         ResourceType.DATA_KEYVAULT: '7.0',
         ResourceType.DATA_STORAGE: '2018-11-09',
         ResourceType.DATA_COSMOS_TABLE: '2017-04-17',
-        ResourceType.MGMT_EVENTHUB: '2017-04-01'
+        ResourceType.MGMT_EVENTHUB: '2017-04-01',
+        ResourceType.MGMT_APPSERVICE: '2018-02-01'
```

Comment:
this should be 2019-03-01? the version with the latest azure-mgmt-web sdk version?

Reply:
Now I am using the version 2019-08-01 for latest profile and 2018-02-01 for hybird 2019-03-01